src/panels/media-browser/hui-dialog-select-media-player.ts

```diff
@@ -55,13 +55,15 @@ export class HuiDialogSelectMediaPlayer extends LitElement {
               "ui.components.media-browser.web-browser"
             )}</paper-item
           >
-          ${this._params.mediaSources.map(
-            (source) => html`
-              <paper-item .itemName=${source.entity_id}
-                >${source.attributes.friendly_name}</paper-item
-              >
-            `
-          )}
+          ${this._params.mediaSources
+            .sort()
```

Comment:
You need to sort by name right? So need a sort function that compares the name

Reply:
Yes?